lib/after.js

```diff
@@ -42,32 +54,41 @@ test.afterEach( function() {
 			return null;
 		}
 
-		driver.getCurrentUrl().then( url => console.log( `FAILED: Taking screenshot of: '${url}'` ), ( err ) => {
-			slackNotifier.warn( `Could not capture the URL when taking a screenshot: '${err}'` );
-		} );
+		driver.getCurrentUrl().then(
+			url => console.log( `FAILED: Taking screenshot of: '${ url }'` ),
+			err => {
+				slackNotifier.warn( `Could not capture the URL when taking a screenshot: '${ err }'` );
+			}
+		);
 
-		filenameCallback = () => `FAILED-${locale}-${screenSize}-${shortTestFileName}-${date}`;
+		filenameCallback = () => `FAILED-${ locale }-${ screenSize }-${ shortTestFileName }-${ date }`;
 	} else if ( config.get( 'saveAllScreenshots' ) === true ) {
-		filenameCallback = () => `${locale}-${screenSize}-${shortDescribeFileName}-${date}-${shortTestFileName}`;
+		filenameCallback = () =>
+			`${ locale }-${ screenSize }-${ shortDescribeFileName }-${ date }-${ shortTestFileName }`;
```

Comment:
why is this on a new line when the one above is on a single line? Is it because of length?